test/e2e/tests/test_scenarios.py

```diff
@@ -128,3 +308,39 @@ class TestScenarios:
     """
     def test_scenario(self, scenario):
         runner.run(scenario)
+
+# assertTags returns AssertError if tags don't match
+def assertTags(expectedTags, latestTags):
+    for expectedTag in expectedTags:
+        tagKeyExist = False
+        for latestTag in latestTags:
+            if expectedTag == latestTag:
+                tagKeyExist = True
+                break
+        if tagKeyExist:
+            continue
+        assert expectedTag == None
+
+# assertParameters returns AssertError if parameters don't match
+def assertParameters(expectedParameters, latestParameters):
+    for expectedParameter in expectedParameters:
+        parameterExist = False
+        for latestParameter in latestParameters:
+            if expectedParameter == latestParameter:
+                parameterExist = True
+                break
+        if parameterExist:
+            continue
+        assert expectedParameter == None
+
+# assertSubnets returns AssertError if subnets don't match
+def assertSubnets(expectedSubnets, latestSubnets):
+    for expectedSubnet in value:
```

Comment:
Did you mean `for expectedSubnet in expectedSubnets` ?

Reply:
Oh yeah, typo there